gulpfile.js

```diff
@@ -152,21 +161,39 @@ function makeDevpackPkg() {
   var webpackStream = require('webpack-stream');
   var webpackConfig = require('./webpack.conf');
   var helpers = require('./gulpHelpers');
+  var updatedModuleList = [];
 
   var cloned = _.cloneDeep(webpackConfig);
   cloned.devtool = 'source-map';
   var externalModules = helpers.getArgModules();
-
   const analyticsSources = helpers.getAnalyticsSources();
   const moduleSources = helpers.getModulePaths(externalModules);
+  updatedModuleList = updateModulesForIH(moduleSources);
 
-  return gulp.src([].concat(moduleSources, analyticsSources, 'src/prebid.js'))
+  return gulp.src([].concat(updatedModuleList, analyticsSources, prebidSrc))
     .pipe(helpers.nameModules(externalModules))
     .pipe(webpackStream(cloned, webpack))
     .pipe(gulp.dest('build/dev'))
     .pipe(connect.reload());
 }
 
+function updateModulesForIH(moduleSources) {
+  var temp = -1;
+  var updatedModuleList = [];
+
+  if (profile == "IH") {
+    var filePathArr = [];
+    updatedModuleList = moduleSources.filter(function(x) {
+      filePathArr = x.split("/");
+      return excludeModules.indexOf(filePathArr[filePathArr.length-1]) < 0;
```

Comment:
excludeModules is used only in this function so we can keep it local to the function

Reply:
done